### ISSUE TITLE
Sprint Review

### ISSUE THREAD

**Issue by libecap** [design-mind/web-app#17]
How did you & the sprint succeed? What was positive?

How did you & the sprint fail? What was negative?

@thomasjbradley

**Comment by libecap** [comment]
**How did you & the sprint succeed? What was positive?**

The sprint succeeded because we worked well as a team. We shared the workload and gave each other feedback. We also meet outside of class every Wednesday for 2 hours to make sure we were on track and to discuss any issues we had will working on our individual sections.

Another way we succeeded was how we selected challenging but achievable sprint goals. We looked for and identified what it would take to achieve this goal and split the work between the three of us. This really helped with time management and checking off tasks.

**How did you & the sprint fail? What was negative?**

We were able to achieve our sprint goals. We had a slow start figuring out our site map and task flow but we were able to complete all the tasks none-the-less  by sharing the workload and meeting together outside of class. I think this will help us throughout the project.

@thomasjbradley

**Comment by FrancescoIaderosa92** [comment]
**How did you & the sprint succeed? What was positive?**

The sprint succeeded by creating more time outside of class to get together and continue to check off stuff in the sprint. There was a lot of positive things to note. Everything was executed and done early.

**How did you & the sprint fail? What was negative?**

The sprint was seen as a success, everything was done and executed in the allotted amount of time when completing the sprint. The only negative thing during this sprint was finding the correct solution when executing a task.

@thomasjbradley

**Comment by JhonTolentino** [comment]
**"How did you & the sprint succeed? What was positive?"**

I succeeded by keeping to the timeline and producing exactly what I promised the team. The topics assigned to me were researched to better help provide a better understanding in the Designer Recruitment Market. The XD file I promised to finish for the team was completed on time, and to what we had agreed on for the look. The biggest positive was the team understood each of our strengths so we designed to within our known capabilities. This allowed us to focus on streamlining flows rather than overcomplicating the app with too many tasks.

**"How did you & the sprint fail? What was negative?"**

I feel we failed in establishing a brand colour and feel as soon as possible. We quickly jumped into what we wanted to do and how we were going to achieve it. We didn't set the branding identity down until mid-week. Luckily everything was still monotonous so merging the branding identity to what we currently had was easy and seamless.

@thomasjbradley
